build-system/tasks/helpers.js

```diff
@@ -426,21 +426,24 @@ async function finishBundle(
     options
   );
 
-  if (options.latestName) {
-    // "amp-foo-latest.js" -> "amp-foo-latest.max.js"
-    const latestMaxName = options.latestName.replace(/\.js$/, '.max.js');
-    // Copy amp-foo-0.1.js to amp-foo-latest.max.js.
+  const logPrefix = options.minify ? 'Minified' : 'Compiled';
```

Comment:
This function still has the old JSDoc. Can you update it with the suggested text?